docs/site/src/demos/lists/InsetList.js

```diff
@@ -0,0 +1,37 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from 'material-ui/List';
+
+const styleSheet = createStyleSheet('InsetList', (theme) => ({
+  root: {
+    width: '100%',
+    maxWidth: '360px',
+    background: theme.palette.background.paper,
+  },
+}));
+
+function SimpleList(props, context) {
+  const classes = context.styleManager.render(styleSheet);
+  return (
+    <List className={classes.root}>
+      <ListItem button>
+        <ListItemText inset primary="Chelsea Otakan" />
```

Comment:
I confirm, the issue is with the icon padding. The indent value is correct.
I think that we need a `ListItemIcon` component to apply the right spacing.
![capture d ecran 2016-12-17 a 20 34 07](https://cloud.githubusercontent.com/assets/3165635/21289255/4513f594-c498-11e6-926b-1d282261208e.png)